docs/sources/installation/helm.md

```diff
@@ -129,7 +129,7 @@ spec:
 
 In order to receive and process syslog messages in Promtail, the following changes will be necessary:
 
-* Review the [Promtail syslog-receiver configuration documentation](/docs/clients/promtail/scraping.md#syslog-receiver)
+* Review the [Promtail syslog-receiver configuration documentation](../../clients/promtail/scraping/#syslog-receiver)
```

Comment:
```suggestion
* Review the [Promtail syslog-receiver configuration documentation](../../clients/promtail/scraping.md#syslog-receiver)
```
Why extra `/` at the end? Still the filename is same `scraping.md` correct?

Reply:
Right, filename is still same. But the url doesn't work with the .md extension as you can test from https://grafana.com/docs/loki/latest/clients/promtail/scraping.md (404) https://grafana.com/docs/loki/latest/clients/promtail/scraping (200)

You can click on https://grafana.com/docs/loki/latest/clients/promtail/scraping#syslog-receiver and see that the last `/` is added to the url automatically before the `#`.

Reply:
Cool. Nice! thanks for clarification! :+1: